src/Symfony/Bundle/FrameworkBundle/Resources/config/routing.xml

```diff
@@ -67,6 +68,7 @@
                 <argument key="matcher_cache_class">%router.cache_class_prefix%UrlMatcher</argument>
             </argument>
             <argument type="service" id="router.request_context" on-invalid="ignore" />
+            <argument type="service" id="parameter_bag" on-invalid="ignore" />
```

Comment:
`on-invalid="ignore"`, because we still allow to install the Fwb 4.x with DI component 3.4